### ISSUE TITLE
Keyboard not working

### ISSUE THREAD

**Issue by KarloX2** [deitch/jstree-grid#256]
Hi!
I'm using jstree-grid V3.9.4 with jstree V3.3.7 and I cannot get the keyboard navigation (arrow keys etc.) to work. More precisely: Only as long as the root element in the tree is selected, the keyboard actions are working. As soon as another node is being selected all the keys are dead and a breakpoint on the "keydown.jstree" handler code doesn't break any more.

However, when I leave out the entire "grid" plugin the keyboard navigation works fine in the entire tree.

Any advice?

Thanks!
Karlo

**Comment by deitch** [comment]
That is a good question. I cannot say that I ever have tried it, with all the years I have spent with jstree. Considering that I am a keyboard shortcut addict, that is pretty funny.

Definitely don't have time to dig in properly. If you can figure it out, would be great.

**Comment by KarloX2** [comment]
Ok. I'm currently trying to find out more.

So far what I see is this: Except for the root element, when I select an element in the tree, afterwards _document.activeElement_ points to the document's body. Without loading the jstree-grid plugin, _document.activeElement_ is always on the _<a class="jstree-anchor"...>_ after selecting something in the tree. I think this makes the difference. But right now I have no clue who/what steals away the focus and makes the body have it.

- Can you confirm this is a general problem (also with an application you have)? Otherwise it may be specific to my app somehow.
- Do you have any idea where I could look at?

Thanks
Karlo

**Comment by deitch** [comment]
Closed via #257

**Comment by KarloX2** [comment]
Super. Can you create a release that will be available via NPM?

**Comment by deitch** [comment]
Already done.

**Comment by KarloX2** [comment]
Perfect, thanks!
